JS/Lesson15/saffet/script.js

```diff
@@ -67,113 +67,348 @@ switch (day) {
+	}
+	else{
+		console.log("You are a senior citizen");
 	}
-
-	// if (age < 18) {
-	// 	console.log('Sorry, you are not old enough to vote');
-	// } else if (age > 65) {
-	//   console.log('You are a citizen');
-	// } else {
-	// 	console.log('You are eligible to vote');
-	// }
 }
 
-checkAge(15);
-checkAge(53);
-checkAge(90);
-checkAge(18);
-checkAge(65);
-
-// 3. Write a program that asks the user to enter two numbers and checks if the first number is greater than, less than, or equal to the second number using if else statement.
+checkAge(17);
+checkAge(43);
+checkAge(78);
+
+console.log("-------------------------------------------------");
+// 3. Write a program that asks the user to enter two numbers and checks if the first number is greater than, less than, 
+//    or equal to the second number using if else statement.
+console.log("Ex3:");
+function numCheck(num1, num2){
+	if(num1<num2){
+		console.log("Number 1 is less then number 2");
+	}
+	else if(num1>num2){
+		console.log("Number 1 is greater then number 2");
+	}
+	else if(num1 === num2){
+		console.log("These numbers are equal");
+	}
+	else{
+		console.log("Not valid comparison");
+	}
+}
+numCheck(3,2);
+numCheck(134,235);
+numCheck(76,76);
 
+console.log("-------------------------------------------------");
 // 4. Write a program that takes in a string and checks if the first letter is uppercase using if else statement.
+console.log("Ex4:");
 
-function checkIfStringIsUpperCase(stringToCheck) {
-	if (stringToCheck[0] === stringToCheck[0].toUpperCase()) {
-		console.log('first letter is uppercase');
-	} else {
-		console.log('is not uppercase');
+function checkUpperCase(string){
+	if(string[0]===string[0].toUpperCase()){
+		console.log("First letter is uppercase");
+	}
+	else{
+		console.log("First letter is not uppercase");
 	}
 }
-
-checkIfStringIsUpperCase('Write a program that takes in');
-checkIfStringIsUpperCase('write a program that takes in');
-
+checkUpperCase("Saffet");
+checkUpperCase("ramazan");
+console.log("-------------------------------------------------");
 // 5. Write a program that takes in a number and checks if it's positive, negative, or zero using if else statement.
+console.log("Ex5:");
 
-// 6. Write a program that takes in a year and checks if it's a leap year (is divisible by 4 and not divisible by 100 OR divisible by 400) or not using if else statement.
+function checkSign(number){
+	if(number < 0){
+		console.log("The number is negative (-)");
+	}
+	else if(number > 0){
+		console.log("The number is positive (+)");
+	}
+	else{
+		console.log("The number is zero (0)");
+	}
+}
+checkSign(3);
+checkSign(-4);
+checkSign(0);
+
+console.log("-------------------------------------------------");
+// 6. Write a program that takes in a year and checks if it's a leap year 
+//(is divisible by 4 and not divisible by 100 OR divisible by 400) or not using if else statement.
+console.log("Ex6:");
+
+function checkLeapYear(year){
+	if(year % 4 == 0 && year % 400 == 0){
```

Comment:
This condition is not correct. Please make sure that the number is **divisible by 4 and not divisible by 100**